sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -193,131 +205,69 @@ public class EventProcessorClient
         ///   to be similar to <c>{yournamespace}.servicebus.windows.net</c>.
         /// </summary>
         ///
-        public string FullyQualifiedNamespace { get; }
+        public new string FullyQualifiedNamespace
```

Comment:
Per conversations with Krzysztof,  we want to maintain binary compatibility with the GA release.  Though the functionality for some members has moved to the base class, they need to continue to be defined here as well with the implementation delegating.